network-api/networkapi/wagtailpages/pagemodels/customblocks/block_with_aside.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.core import blocks
+from .aside_fields import aside_fields
+from .listing import ListingBlock
+from .full_content_rich_text_options import full_content_rich_text_options
+
+
+class BlockWithAside(blocks.StructBlock):
+    content = blocks.StreamBlock([
+        ("listing", ListingBlock()),
+        ('paragraph', blocks.RichTextBlock(
+            features=full_content_rich_text_options
+        ))
+    ], max_num=1, help_text='The wider block that appears on the left on desktop', icon='doc-full')
```

Comment:
What do you thing about using the same indentation for all the attributes? I think that is a bit more common. 


I know we need to set up a formatter for this repo ... 😓

Reply:
I've pushed the attributes to a new line now so they can be at the same indent, hopefully it's closer to what you were expecting now.